typescript-local-storage/data.ts

```diff
@@ -0,0 +1,18 @@
+/* exported todos writeTodos */
+interface Todo {
+  todoId: string;
+  task: string;
+  isCompleted: boolean;
+}
+
+let todos: Todo[] = readTodos();
+
+function writeTodos(): void {
+  const todosJSON: string = JSON.stringify(todos);
+  localStorage.setItem('todos-storage', todosJSON);
+}
+
+function readTodos(): Todo[] {
+  const jsonString = localStorage.getItem('todos-storage') as string;
+  return JSON.parse(jsonString);
+}
```

Comment:
What happens if you don't have anything in local storage here? (You would be seeing errors in your console).

You don't want to be asserting a type here because you are guaranteeing that there is a string in local storage, which there will not be a string there is a user has never visiting the application before.